build-system/tasks/karma.conf.js

```diff
@@ -40,6 +41,14 @@ const BABELIFY_CONFIG = Object.assign({}, BABELIFY_GLOBAL_TRANSFORM, {
   sourceMapsAbsolute: true,
 });
 
+// minify-replace babel plugin used to replace experiment constants
+const replacePlugin = conf.getReplacePlugin();
+if ('plugins' in BABELIFY_CONFIG) {
```

Comment:
nit: From what I understand, `BABELIFY_CONFIG['plugin']` should never exist here. @erwinmombay @estherkim correct me if that' not correct.

Reply:
Currently no, just added this check in case future configs contain existing plugins

Reply:
I think that's correct - there isn't any right now.

Reply:
nit: create a const in `build-system/tasks/helpers.js` similar to `BABELIFY_GLOBAL_TRANSFORM` and add to the `Object.assign()` line when building the babelify options
```
const babelifyOptions = Object.assign({}, BABELIFY_GLOBAL_TRANSFORM, BABELIFY_REPLACE_PLUGIN);
```

Then you can reuse this when building the babelify options in `karma.conf.js`